graylog2-server/src/main/java/org/graylog2/shared/rest/resources/system/RemoteDeflectorResource.java

```diff
@@ -20,6 +20,6 @@
 import retrofit2.http.POST;
 
 public interface RemoteDeflectorResource {
-    @POST("/system/deflector/cycle")
+    @POST("system/deflector/cycle")
```

Comment:
Are these changes strictly necessary, or rather can we catch this in a central place?
I'm worried that plugins that implement a ProxiedResource will forget to make the path relative and thus break at runtime.